app/src/main/res/values/donottranslate.xml

```diff
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="package_name">org.fossify.contacts</string>
+    <string name="package_name" translatable="false">org.fossify.contacts</string>
     <string name="app_name">Fossify Contacts</string>
```

Comment:
Might as well mark `app_name` with `translatable="false"`

Reply:
If you agree to such an approach, yes. The app_name should be marked as well.